service/src/main/kotlin/no/nav/su/se/bakover/service/statistikk/mappers/BehandlingStatistikkMapper.kt

```diff
@@ -0,0 +1,319 @@
+package no.nav.su.se.bakover.service.statistikk.mappers
```

Comment:
Det er mulig å bruke `git mv` for å slippe få deleted/created

Reply:
Tror kanskje denne skiftet navn, ble flyttet, og fikk endret innhold ganske omfattende, så mulig det var derfor. Ser f.eks. ut som StønadStatistikkMapper ble moved. Uansett; I blame IntelliJ.